src/pages/documentation/release-notes/index.md

```diff
@@ -0,0 +1,399 @@
+---
+description: Release notes and change logs for the Adobe Experience Platform Mobile SDKs.
+---
+
+# Release notes
+
```

Comment:
Some content is missing here June 30 -> Aug 2

Reply:
Yup! I'll re-add this once I do my last checkover time.